samples/winapi_filefind/main.c

```diff
@@ -16,7 +16,7 @@ int mount_drive_c ()
     if (!NT_SUCCESS(status)) {
         debugPrint("Failed to mount C: drive!\n");
         Sleep(5000);
-        return -1;
+        return 1;
```

Comment:
I'd keep this at `-1` and instead of returning `ret` from main (after checking result from `mount_drive_c`) return `1` regardless.

That is cleaner because in C world (outside of `main`) we'd also expect negative numbers for errors. Someone might extend this function with more return codes in the future, so it shouldn't be tied directly to `main` in the first place.

Reply:
I've changed it to return -1 in the mount_drive_c function and return 1 from main on failure.